smarts/sstudio/generators.py

```diff
@@ -20,6 +20,7 @@
 import os
 import random
 import logging
+from smarts.core.utils.file import make_dir_in_smarts_log_dir
```

Comment:
This line should be in the 3rd part of imports.

Reply:
Sorry, I have automatic imports that tend to throw the imports in random places. :facepalm:

Reply:
Seems you have a new tool to help re-order the imports. I like that re-ordering.